agent/consul/state/intention.go

```diff
@@ -838,23 +839,14 @@ func (s *Store) legacyIntentionMatchTxn(tx ReadTxn, ws memdb.WatchSet, args *str
 //
 // The returned intentions are sorted based on the intention precedence rules.
 // i.e. result[0] is the highest precedent rule to match
-func (s *Store) IntentionMatchOne(
-	ws memdb.WatchSet,
-	entry structs.IntentionMatchEntry,
-	matchType structs.IntentionMatchType,
-) (uint64, structs.Intentions, error) {
+func (s *Store) IntentionMatchOne(ws memdb.WatchSet, entry structs.IntentionMatchEntry, matchType structs.IntentionMatchType, destinationType structs.IntentionTargetType) (uint64, structs.Intentions, error) {
```

Comment:
I'm a bit partial to the other style but that's just personal preference 🤷🏼 

do we have a team stance here?

Reply:
I'm not sure I understand what you mean by "the other style"?